README.md

```diff
@@ -50,12 +50,12 @@ npm install @dsnp/parquetjs
 ```
 
 ```typescript
-import { AnnouncementType } from "@dsnp/schemas"; 
+import { AnnouncementType } from "@dsnp/schemas";
 import { parquet } from "@dsnp/schemas";
-import { ParquetWriter } from "@dsnp/parquetjs";
+import { ParquetWriter, ParquetSchema } from "@dsnp/parquetjs";
 
 const [parquetSchema, writerOptions] = parquet.fromDSNPSchema(descriptorForAnnouncementType(AnnouncementType.Broadcast).parquetSchema);
-const writer = await ParquetWriter.openFile(parquetSchema, "./file.parquet", writerOptions);
+const writer = await ParquetWriter.openFile(new ParquetSchema(parquetSchema), "./file.parquet", writerOptions);
```

Comment:
This is the change we should highlight in the changelog. Effectively instead of constructing this class object in here, we require devs to do it outside. Not a bad tradeoff for dropping the dependency.